demo/test/snapshots/linting-config.test.js.md

```diff
@@ -88,9 +88,6 @@ Generated by [AVA](https://ava.li).
         "@typescript-eslint/keyword-spacing": [␊
           "off"␊
         ],␊
-        "@typescript-eslint/lines-around-comment": [␊
```

Comment:
@FSVetaz: I understand running this command, but I am extremely confused as to why this change does not show anywhere in the resulting files changed.

Reply:
With or without it, nothing changes in the resulting eslint linting/formatting